invitation/client.go

```diff
@@ -76,6 +76,42 @@ func (c *Client) Create(ctx context.Context, params *CreateParams) (*clerk.Invit
 	return invitation, err
 }
 
+type BulkCreateParams struct {
+	clerk.APIParams
+	Invitations []*CreateParams
+}
+
+func (b BulkCreateParams) MarshalJSON() ([]byte, error) {
+	return json.Marshal(b.Invitations)
+}
+
+type BulkCreateResponse struct {
+	clerk.APIResource
+	Invitations []*clerk.Invitation
+}
+
+func (b *BulkCreateResponse) UnmarshalJSON(data []byte) error {
+	return json.Unmarshal(data, &b.Invitations)
+}
+
+// BulkCreate creates multiple invitations.
+func (c *Client) BulkCreate(ctx context.Context, params *BulkCreateParams) (*clerk.InvitationList, error) {
+	req := clerk.NewAPIRequest(http.MethodPost, fmt.Sprintf("%s/bulk", path))
+	req.SetParams(params)
+
+	res := &BulkCreateResponse{}
+	err := c.Backend.Call(ctx, req, res)
+	if err != nil {
+		return nil, err
+	}
+
+	return &clerk.InvitationList{
```

Comment:
Similar to [this thread](https://github.com/clerk/clerk-sdk-go/pull/351#discussion_r1838247796), this can actually be improved if we introduce another type. However, because this cannot be paginated because it's a response, it’s not under a key, it’s just an array. So we would lose the `APIResource` in the process.